### ISSUE TITLE
ionic 4 angular 8

### ISSUE THREAD

**Issue by sylwester-** [RaschidJFR/ionic-header-parallax#11]
Hello,

Fresh install, I get

```
[ng] ERROR in ./src/app/app.module.ts 18:18-43
[ng] "export 'IonicHeaderParallaxModule' was not found in 'ionic-header-parallax'
```

```
Could not find a declaration file for module 'ionic-header-parallax'. '<app root>/node_modules/ionic-header-parallax/index.js' implicitly has an 'any' type.
  Try `npm install @types/ionic-header-parallax` if it exists or add a new declaration (.d.ts) file containing `declare module 'ionic-header-parallax';`ts(7016)
```

Thanks

**Comment by RaschidJFR** [comment]
thanks for reporting. What’s your ionic info?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by szepgeorge** [comment]
Same here ..

**Comment by RaschidJFR** [comment]
Mmm... i've just tested a fresh install with Angular 7 and it works so there must be something related to v8. I'll run some tests later today.

**Comment by sylwester-** [comment]
I agree, probably ng 8. Really looking forward trying it out. 

Looks very nice

**Comment by RaschidJFR** [comment]
I had a quick look at the project. It looks like it’s just the declaration file missing (it was never created before), not really an Ng8 issue. 

I think this may already be fixed in the latest commit but it is not yet published to npm. As a workaround could you please try [this  solution from stackoverflow](https://stackoverflow.com/a/50516783/3000466)?

If any of you want to give me a hand you could download the repo and run it with your local version of Angular 8. Just let me know if it works ok so I can release it to npm, as I’m currently using Angular7 and I’m in the middle of some job so I wouldn’t like to mess with my angular version just now. The instructions for running the repo locally are in the [CONTRIBUTING.md](https://github.com/RaschidJFR/ionic-header-parallax/blob/master/CONTRIBUTING.md) file.

**Comment by RaschidJFR** [comment]
Fixed in baa0157ed83050bc134a4d9c04a37b41f9d66e4c. Published in v2.2.2
